index.js

```diff
@@ -108,10 +108,10 @@ Manager.prototype.reconnect = function(url){
   }
 
   this.socket = new Socket(url, opts);
-  this.socket.onopen = this.onOpen.bind(this);
-  this.socket.onclose = this.onClose.bind(this);
-  this.socket.onmessage = this.onMessage.bind(this);
-  this.socket.onerror = this.onError.bind(this);
+  this.socket.on('open', this.onOpen);
+  this.socket.on('close', this.onClose);
+  this.socket.on('message', this.onMessage);
```

Comment:
@rauchg Is this usage not correct? https://github.com/cloudup/mydb-client/blob/6064de4b9da5cbb170328ac5102594481f2085b0/index.js#L154-L155